multimodal/tests/unittests/others_2/test_ner_chinese.py

```diff
@@ -75,12 +75,16 @@ def bio_samples_to_df(samples):
 
 
 def get_data():
-    train_data = open(os.path.join(get_home_dir(), "dev.txt")).read()
+    train_file_path = os.path.join(get_home_dir(), "train.txt")
```

Comment:
Also fixed this issue in another PR: #4737

Reply:
Ok

Reply:
Close this PR